content/redirects/how-federalist-works-for-presentation.html

```diff
@@ -0,0 +1,5 @@
+---
+permalink: /assets/documents/how-federalist-works-for-presentation.pdf/index.html
+redirect: /pages/documentation/how-pages-works/
```

Comment:
should this redirect to the PDF you're including at: 

_assets/documents/how-pages-works-diagram.pdf

Reply:
I'm not sure honestly.  We can delete how-pages-works-diagram.pdf and redirect to /pages/documentation/how-pages-works/.  For how-federalist...pdf, this was just a best guess.

Reply:
ah, federalist is the old name of Pages, so i think it has just been replaced by newer content 2x